requirements-rocm.txt

```diff
@@ -7,9 +7,9 @@ ray >= 2.9
 sentencepiece  # Required for LLaMA tokenizer.
 numpy
 tokenizers>=0.15.0
-transformers >= 4.39.0  # Required for StarCoder2.
+transformers >= 4.39.1  # Required for StarCoder2 & Llava.
 fastapi
 uvicorn[standard]
 pydantic >= 2.0  # Required for OpenAI server.
 prometheus_client >= 0.18.0
-outlines == 0.0.34
\ No newline at end of file
+outlines == 0.0.34
```

Comment:
What’s the change here?

Reply:
I actually have no idea why this showed up.. will take a look and fix it

Reply:
Ah! This file didn't have an empty last line like other requirements.txt file so my editor added it automatically when I pushed the change